signac/contrib/collection.py

```diff
@@ -938,6 +938,35 @@ def dump(self, file=sys.stdout):
         else:
             self._dump(file)
 
+    def to_json(self, file=None):
+        """Dump the collection as a JSON file.
+
+        :param file: The file to write the JSON string to
```

Comment:
@csadorf @bdice do we have a standard for docstrings for functions that sometimes (but don't always) return values?

Reply:
No, we don't.